docs/conf.py

```diff
@@ -51,9 +47,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.3'
+version = pkg_resources.get_distribution("django-silk").version
 # The full version, including alpha/beta/rc tags.
-release = '0.3'
+release = version
```

Comment:
Let's use the way release and version is distinguished in the setuptools-scm docs; https://github.com/pypa/setuptools_scm/#usage-from-sphinx

Reply:
Fixed.